engine/Cuboid.js

```diff
@@ -0,0 +1,68 @@
+class Cuboid {
+  constructor(width, height, depth) {
+    this.width = width;
+    this.height = height;
+    this.depth = depth;
+  }
+
+  get vertices() {
+    let cubeVertices = [
+      // vordere Fläche
+      -0.5, -0.5,  0.5,
+      0.5, -0.5,  0.5,
+      0.5,  0.5,  0.5,
+      -0.5,  0.5,  0.5,
+
+      // hintere Fläche
+      -0.5, -0.5, -0.5,
+      -0.5,  0.5, -0.5,
+      0.5,  0.5, -0.5,
+      0.5, -0.5, -0.5,
+
+      // obere Fläche
+      -0.5,  0.5, -0.5,
+      -0.5,  0.5,  0.5,
+      0.5,  0.5,  0.5,
+      0.5,  0.5, -0.5,
+
+      // untere Fläche
+      -0.5, -0.5, -0.5,
+      0.5, -0.5, -0.5,
+      0.5, -0.5,  0.5,
+      -0.5, -0.5,  0.5,
+
+      // rechte Fläche
+      0.5, -0.5, -0.5,
+      0.5,  0.5, -0.5,
+      0.5,  0.5,  0.5,
+      0.5, -0.5,  0.5,
+
+      // linke Fläche
+      -0.5, -0.5, -0.5,
+      -0.5, -0.5,  0.5,
+      -0.5,  0.5,  0.5,
+      -0.5,  0.5, -0.5
+    ];
```

Comment:
Or would you rather like to get back a `Float32Array`?

Reply:
It does not really matter where you convert it to a `Float32Array` (obviously 🙂). My idea of making it directly a `Float32Array` was that you don't have to care whether you have to convert it or not at a later stage…
And a `Float32Array` has all methods a normal Array has so it does not really make a difference.

Reply:
Ok. Then I prefer this solution. Because it makes it easier to introduce new meshes.